mobile/lib/modules/asset_viewer/providers/render_list.provider.dart

```diff
@@ -16,6 +16,15 @@ final renderListProvider =
   );
 });
 
+final renderListProviderWithGrouping =
```

Comment:
Nit. Maybe we can migrate the `renderListProviderWithGrouping` and the existing `renderListProvider` to the riverpod_generator and named argument for `GroupAssetsBy`? We can leave them as it is and can always migrate them later as well

Reply:
let's migrate later :-)